src/utils/neurons.utils.ts

```diff
@@ -15,6 +15,7 @@ const voteForProposal = ({
   proposalId: ProposalId | undefined;
 }): Vote | undefined => {
   if (!proposalId) {
+    console.error("VFP0", recentBallots);
```

Comment:
I assume this is intentional for debugging user problems?  I'd be curious how this could be called without a proposal ID but that is outside the scope of this PR.

Reply:
thanks, removed